src/utils.php

```diff
@@ -32,3 +32,15 @@ function pathToUri(string $filepath): string {
     $filepath = implode('/', array_map('urlencode', explode('/', $filepath)));
     return 'file:///' . $filepath;
 }
+
+/**
+ * Transforms URI into file path
+ * 
+ * @param string $uri
+ * @return string
+ */
+function uriToPath(string $uri)
+{
+    $filepath = urldecode(parse_url($uri)['path']);
```

Comment:
A URI can have any protocol, but only file URIs can be mapped to a path. We need to throw an `InvalidArgumentException` if the URI is not a `file://` URI and also handle/test that case in the implementation.